src/sqlfluff/core/templaters.py

```diff
@@ -66,7 +68,7 @@ def __init__(self, **kwargs):
         """
 
     @staticmethod
-    def process(in_str, fname=None, config=None):
+    def process(*, in_str, fname=None, config=None):

```

Comment:
Maybe that's controversial but I wanted to force the use of `keyword arguments` so I could swap the `in_str` and `fname` in the DbtTemplater `process` function signature https://github.com/sqlfluff/sqlfluff/pull/508/files#diff-0f428d02c08ceb820ea8ef225ad9684548d08a6259262743253706d01e1a5953R518

This is because the DbtTemplater does not support an `in_str` so `fname` is required

Reply:
Ahhhh. Understood. I'm ok with that. Could you add a note in the docstring to that effect. It's quite a subtle point and may not be immediately obvious to other users.

Reply:
done now :) I should have added it in the first place, thanks for pointing out